src/modules/ExlMd2Html.js

```diff
@@ -62,7 +63,7 @@ function converter(mdString) {
   const hast = h('html', [
     h('body', [
       h('header', []),
-      h('main', [h('div', content.hast)]),
+      h('main', content.hast),
```

Comment:
I see that this div is anyway stripped out in the franlin output. Danaher project too follow this structure.

https://github.com/hlxsites/danaher-ls-aem/blob/7c3ed992c522da25a0f6d5dd94a9f33a2e6aecf2/tools/actions/convert/src/modules/md2html.js#L57

Reply:
However, doing this resulting in content loss (half the markup is lost when it's out of franklin).
We have to be little cautious about this change I guess
https://258616-518tealdonkey.adobeioruntime.net/api/v1/web/main/convert/docs/authoring-guide-exl/using/markdown/cheatsheet
https://main--exlm--sivaramvt.hlx.page/docs/authoring-guide-exl/using/markdown/cheatsheet